pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -19,8 +19,11 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormGroup, HelpBlock, Modal, TypeAheadSelect } from 'patternfly-react';
-import { Button, Tooltip, TooltipPosition } from '@patternfly/react-core';
+import { FormGroup, HelpBlock, Modal } from 'patternfly-react';
+import {
+    Select as SelectTypeAhead, SelectOption, SelectVariant,
```

Comment:
Why the SelectTypeAhead aliasing? IMHO that's rather confusing when reading the code and trying to map it to the PF4 documentation. I see it's needed to disambiguate from cockpit-components-select, but maybe call them PFSelect and CockpitSelect?

Reply:
I needed the Aliasing because we already include the Select component from another file. I will replace the strings to the suggested naming.